sdk/eventhub/Azure.ResourceManager.EventHubs/README.md

```diff
@@ -1,69 +1,51 @@
-# README.md template
+# Azure Management EventHubs client library for .NET
```

Comment:
nit:  There should be a space in the product name:  `Event Hubs`.    Leaving one client here, but the same is true for the other non-code references below.

Reply:
Got it, thanks